### DISCUSSION TITLE
cd ios not found

### DISCUSSION THREAD

**Discussion by senkyte** [sst-inc/help#52] [Q&A - React] [answered]
help, im stuck at when the pod install section, and im unable to submit on time. whenever i try typing cd ios, it says that there is no such file or directory

**Comment by ghost** [comment]
Can you send some sort of screenshot of your Terminal?

**Reply by senkyte** [reply]
<img width="340" alt="Screenshot 2021-10-30 at 11 59 41 PM" src="https://user-images.githubusercontent.com/87954731/139540481-eff8cbf5-31fb-4d7b-9ffe-3d6cb7966298.png">

**Reply by Ethan-Chew** [reply]
Could you run 'ls' in your terminal?

**Comment by senkyte** [comment]
<img width="466" alt="Screenshot 2021-10-31 at 4 35 14 PM" src="https://user-images.githubusercontent.com/87954731/139574695-b705a56f-f4b7-4294-adab-4b1d7cce3cb8.png">

**Reply by ghost** [reply]
You are not in your TasksApp folder. Try to cd to your TasksApp folder.

**Comment by senkyte** [comment]
they say no such file or directory

**Comment by senkyte** [comment, marked as answer]
WAIT NVM I THINK I SOLVED IT BY DOING CD DOWNLOADS FINALLY

**Comment by senkyte** [comment]
shit xcode update
